sites/demo/requirements/base.txt

```diff
@@ -4,5 +4,5 @@ django-storages==1.10.1
 dockerflow==2020.10.0
 gunicorn==20.0.4
 psycopg2-binary==2.8.6
-richie==2.0.0-beta.21
+richie==2.0.0-beta.22
 sentry-sdk==0.19.1
```

Comment:
can you add factory as a base dependency?

Reply:
Ah ah 😅
https://github.com/openfun/richie-site-factory/commit/d72e014bb8e38de3dbf5b7ac4b0d6d1a7f0cbc4d